src/Symfony/Component/Validator/Constraints/Timezone.php

```diff
@@ -46,6 +48,15 @@ class Timezone extends Constraint
      */
     protected static $errorNames = self::ERROR_NAMES;
 
+    /**
+     * @param int|array<string,mixed>|null $zone           Restrict valid timezones to this geographical zone(s) (see {@see \DateTimeZone}), or this constraint's options as an associative array
+     * @param string|null                  $message        The message if the validation fails
+     * @param string|null                  $countryCode    Restrict the valid timezones to this country if the <pre>zone</pre> option is <pre>\DateTimeZone::PER_COUNTRY</pre>
```

Comment:
I don't think we want to put HTML in docblocks.

```suggestion
     * @param string|null                  $countryCode    Restrict the valid timezones to this country if the zone option is {@see \DateTimeZone::PER_COUNTRY}
```

Reply:
Ok, and what about code samples? For example in [File](https://github.com/squrious/symfony/blob/05047c5c1a0d7104a334fe6645ace3c363c64af8/src/Symfony/Component/Validator/Constraints/File.php#L104) I used a `pre` tag to provide an array example. Without it it's quite unreadable. Is it better to not provide the example at all?

Reply:
I would not put code snippets in the phpdoc. If the explanation involves code snippets, it deserves being in the symfony documentation IMO (which then allows to have the code snippet for all the mapping formats)

Reply:
I put a link to the documentation